roles/openshift_logging_mux/defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 ### General logging settings
-openshift_logging_mux_image_prefix: "{{ openshift_hosted_logging_deployer_prefix | default('docker.io/openshift/origin-') }}"
-openshift_logging_mux_image_version: "{{ openshift_hosted_logging_deployer_version | default('latest') }}"
+openshift_logging_mux_image_prefix: "{{ openshift_logging_image_prefix | default(__openshift_logging_image_prefix) }}"
```

Comment:
Do you think its worth considering support for running one version of the fluentd image for mux and another for the collectors?  Wouldn't we be concerned about the testing matrix this might introduce?